apps/spruce/src/components/SpruceForm/Widgets/DayPicker.tsx

```diff
@@ -0,0 +1,30 @@
+import { Description, Label } from "@leafygreen-ui/typography";
+import { DayPicker } from "components/DayPicker";
+import ElementWrapper from "../ElementWrapper";
+import { SpruceWidgetProps } from "./types";
+
+export const DayPickerWidget: React.FC<
+  {
+    options: {
+      format?: string;
+    };
```

Comment:
was there anything you wanted to do with the `format` arg in this PR? Just curious!

Reply:
copy🍝